keycloakClient/api.go

```diff
@@ -24,11 +26,19 @@ type Client struct {
 	tokens           map[string]*TokenInfo
 }
 
+// LoggingClient
+type LoggingClient struct {
+	stdClient http.Client
```

Comment:
hey hey, if I give you pseudo code, I don't wanna see it committed, you should know that CTO code isn't production level code ;-) Prefer `standardClient` or `defaultClient`, but none of this ambiguous/abbreviated variable naming stuff

Reply:
I got too excited, i am commenting things right now